app/helpers/cloud_volume_helper/textual_summary.rb

```diff
@@ -61,6 +61,12 @@ def textual_storage_resource
     nil
   end
 
+  def textual_storage_service
+    @record.ext_management_system.storage_services.find(@record.storage_service_id)
```

Comment:
If you have an ID can't you just do `StorageService.find(@record.storage_service_id)` or better yet do you have `@record.storage_service` ?